lib/SingleSelleckt.js

```diff
@@ -189,21 +189,38 @@ _.extend(SingleSelleckt.prototype, {
         matchTerm = term.toLowerCase();
 
         return _.reduce(items, function(memo, item){
-            var matchIndex;
+            var matchIndex, matchStart, matchEnd;
 
             if (!item.value) {
                 return memo;
             }
 
             matchIndex = item.label.toLowerCase().indexOf(matchTerm);
+            matchStart = matchIndex;
+            matchEnd = matchIndex + (term.length - 1);
+
+            if (matchIndex === -1 && item.aliases && item.aliases.length > 0) {
+                // try the aliases
+                matchIndex = item.aliases.reduce(function(index, alias) {
+                    if (index === -1) {
+                        return String(alias).toLowerCase().indexOf(matchTerm);
+                    } else {
+                        return index;
+                    }
+                }, -1);
+
+                if (matchIndex !== -1) {
+                    matchStart = matchEnd = 0;
```

Comment:
I wanted the values to be `.substr()`-safe. But I've checked the way both variables are used in the code and seems like 0 and -1 would work the same way — no substring highlighting will happen in the dropdown. I'll change them both to -1 as this is clearly a special case.